node/src/command.rs

```diff
@@ -122,9 +122,9 @@ impl SubstrateCli for Cli {
     fn description() -> String {
         format!(
             "Manta/Calamari/Dolphin Collator\n\nThe command-line arguments provided first will be \
-		passed to the parachain node, while the arguments provided after -- will be passed \
-		to the relaychain node.\n\n\
-		{} [parachain-args] -- [relaychain-args]",
+        passed to the parachain node, while the arguments provided after -- will be passed \
```

Comment:
not sure reformatting these multiline strings is a good idea, the default tab width in bash is 8, not 4 so this will visually move the CLI

Reply:
nevermind, leading space is discarded from nonliteral strings